docs/miscellaneous.rst

```diff
@@ -15,6 +15,8 @@ Statically-sized variables (everything except mapping and dynamically-sized arra
 - If an elementary type does not fit the remaining part of a storage slot, it is moved to the next storage slot.
 - Structs and array data always start a new slot and occupy whole slots (but items inside a struct or array are packed tightly according to these rules).
 
+The the contract uses inheritance, the ordering is determined as the C3-linearized order of contracts starting with the most base-ward contract.
```

Comment:
Is "The the" what you've meant?

Reply:
f797d93e827d328a88360e903af6e09db0dee76f